### ISSUE TITLE
Problems uploading transactions by CSV

### ISSUE THREAD

**Issue by SJ-bond** [younginnovations/aidstream-old#153]
Hello, some organisations are having problems uploading transactions by CSV. See this email from BRAC for example:

I looked into the transactions myself and noticed those without transaction types. I double-checked our .CSV and they seem normal – data in proper columns. Still after uploading some data goes without transaction type and some other mixes up.

It is strange, because it is like half of the same file will upload normally and only some lines will get messed up.
I noticed that might do with formatting the actual numbers - when there is “0” zero for entire period of transactions(Income, Disbursements, Expenditure), it messes up all of their types and on AIDSTREAM it shows as if amount was left blank.
If only one transaction is “zero” for the period, it will not mess up the types. Still when you view the “0” entry in EDIT screen, it will show nothing in the amount box.
If there is any other number, regardless of what, AIDSTREAM will accept the formatting.
It would be helpful if AIDSTREAM returns a notification when entries do not get type or amount entry. It does for example bring up red message if date is not in date format but let’s say general number.
To summarise – in the VIEW screen, if transaction looks as”.00” – there is no amount in the EDIT screen.
If in the VIEW screen, transaction looks as “0.00” – there is “0” in amount cell in EDIT screen.
